### ISSUE TITLE
Missing label/control associations in "New script" dialog for accessibility

### ISSUE THREAD

**Issue by arantius** [greasemonkey/greasemonkey#1010]
Originally: [DevjaVu ticket 116](http://greasemonkey.devjavu.com/ticket/116)

In XUL files, the control attribute is used on label elements to associate them with the control (for example textbox) they're labelling. In GreaseMonkey?, these are missing from the "New script" dialog XUL file. Screen readers for the blind cannot automatically associate labels to controls, so XUL has to provide that information, or the blind person will not know what a textbox is for.
